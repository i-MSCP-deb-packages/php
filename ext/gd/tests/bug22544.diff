001+ e07677370915ca6e7d74b0cd8db78d3a
001- 10a57d09a2c63fad87b85b38d6b258d6