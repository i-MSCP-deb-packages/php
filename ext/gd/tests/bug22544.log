
---- EXPECTED OUTPUT
10a57d09a2c63fad87b85b38d6b258d6
---- ACTUAL OUTPUT
e07677370915ca6e7d74b0cd8db78d3a
---- FAILED
