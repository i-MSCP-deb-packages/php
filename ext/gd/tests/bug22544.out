e07677370915ca6e7d74b0cd8db78d3a